src/main/java/frc/robot/subsystems/PositionAngleConverter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package frc.robot.subsystems;
+
+/**
+ *
+ * @author dcowden
+ */
+public class PositionAngleConverter {
+
+    public static final int ENCODER_CLICKS_PER_HOOD_MOTOR_REVOLUTION = 4;
+    public static final int HOOD_GEAR_RATIO = 4;
+    public static final int HOME_OFFSET_COUNTS = 50;
+    
+    
+    public double positionFromAngle(double angle) {
+        return -(Math.abs((angle / 360) * ENCODER_CLICKS_PER_HOOD_MOTOR_REVOLUTION * HOOD_GEAR_RATIO - HOME_OFFSET_COUNTS));
+    }
+    public double angleFromPosition(double position){
+        return 0;
+    }
+    
```

Comment:
I like that it has been moved to seperate class. I can write tests tommorow in my free time.

Reply:
true-- but now i've realized do we even need this class? is it possible set up your vision tracking formula so that we always just work in encoder clicks? that way all this code and math goes away